scripts/web-platform-release-notes.js

```diff
@@ -162,13 +194,21 @@ async function main() {
     `Preparing the beta release notes for ${nextBetaVersion} (to be released on ${nextBetaReleaseDate}).`
   );
 
+  const branchName = BRANCH_NAME_PREFIX + nextBetaVersion;
+
   // --------------------------------------------------
-  // 2. Check if there isn't already a release notes draft for the next beta version.
+  // 2. Check if there isn't already a published or draft release notes for the next beta version.
   // --------------------------------------------------
 
-  const rawFileResponse = await fetch(getReleaseNoteRawMDFilePath(nextBetaVersion));
-  if (rawFileResponse.status === 200) {
-    console.error(`A PR is already open for the next beta release notes. File exists: ${getReleaseNoteMDFilePath(nextBetaVersion)}.`);
+  const alreadyExists = await releaseNotesAlreadyExists(nextBetaVersion);
+  if (alreadyExists) {
+    console.error(`Release notes for the next beta version ${nextBetaVersion} already exist.`);
+    process.exit(0);
+  }
+
+  const draftAlreadyExists = await releaseNotesDraftAlreadyExists(nextBetaVersion, branchName);
+  if (draftAlreadyExists) {
+    console.error(`A draft release notes for the next beta version ${nextBetaVersion} already exist on the ${branchName} branch.`);
```

Comment:
```suggestion
    console.error(`Draft release notes for the next beta version ${nextBetaVersion} already exist on the ${branchName} branch.`);
```
alts:
Draft release notes already exist
A draft of the release notes already exists